wsd/COOLWSD.cpp

```diff
@@ -4075,30 +4075,42 @@ int COOLWSD::innerMain()
     JailUtil::cleanupJails(CleanupChildRoot);
 #endif // !MOBILEAPP
 
-    const int returnValue = UnitBase::uninit();
-
-    LOG_INF("Process [coolwsd] finished with exit status: " << returnValue);
-
-    SigUtil::addActivity("finished with status " + std::to_string(returnValue));
-
-    if constexpr (Util::isMobileApp())
-        Util::forcedExit(returnValue);
-
-    return returnValue;
-#else // IOS
-    return 0;
-#endif
+#endif // !IOS
 }
 
 std::shared_ptr<TerminatingPoll> COOLWSD:: getWebServerPoll ()
 {
     return WebServerPoll;
 }
 
-void COOLWSD::cleanup([[maybe_unused]] int returnValue)
+int COOLWSD::cleanup(int returnValue)
 {
+    if constexpr (Util::isMobileApp())
```

Comment:
this shouldn't be done for `IOS` (for which `Util::isMobileApp()` is true AFAIU)